inc/thorvg.h

```diff
@@ -1219,9 +1219,10 @@ class TVG_API Picture final : public Paint
     /**
      * @brief Loads a raw data from a memory block with a given size.
      *
-     * @warning Please do not use it, this API is not official one. It could be modified in the next version.
+     * @retval Result::Success When succeed, Result::InsufficientCondition otherwise.
+     * @retval Result::NonSupport When trying to load a file with an unknown extension.
```

Comment:
@hermet if you do not agree with #1434 please change FailedAllocation to NonSupport in the description below:
```
     * @retval Result::Success When succeed.
     * @retval Result::InvalidArguments In case no data are provided or the @p width or @p height value is zero or less.
     * @retval Result::FailedAllocation When problem with memory allocation occurs.
     * @retval Result::InsufficientCondition If an error occurs at a later stage.
```

Reply:
@hermet #1435 ? feel free to close #1435  if that's not a good time to promote the corresponding Capis.

Reply:
It's time to go. Instead we need to add test code also.

Reply:
here it comes. https://github.com/thorvg/thorvg/pull/1429/commits/fdec1468e5259e0f4778f0f6f54f632b2dad8c6b

Reply:
https://github.com/thorvg/thorvg/pull/1436
tests++